compose/docker-compose.yml

```diff
@@ -38,13 +38,19 @@ services:
       - "127.0.0.1:62001:3306"
 
   elasticsearch:
-    image: "elasticsearch:1.7-alpine"
-    command: "elasticsearch -Des.node.name=TestNode -Des.network.host=0.0.0.0"
-    privileged: yes
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.4.1
+    environment:
+      - cluster.name=am-cluster
+      - node.name=am-node
+      - network.host=0.0.0.0
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
     volumes:
       - "elasticsearch_data:/usr/share/elasticsearch/data"
-    expose:
-      - "9200"
```

Comment:
Would you be able to make the same change in the other services for consistency?

Reply:
Will do ;)